crates/turbo-tasks-memory/src/memory_backend.rs

```diff
@@ -118,54 +121,58 @@ impl MemoryBackend {
         self.memory_tasks.get(*id).unwrap()
     }
 
-    pub fn on_task_might_become_inactive(&self, task: TaskId) {
-        if let Some(gc_queue) = &self.gc_queue {
-            gc_queue.task_might_become_inactive(task);
-        }
-    }
-
-    pub fn on_task_flagged_inactive(&self, task: TaskId, compute_duration: Duration) {
+    pub fn run_gc(
```

Comment:
It's not entirely clear to me what the return value here is supposed to represent. A comment might be nice (or maybe an enum with a clear type name instead of a `bool`?)